modules/apps/josh-lapurd-theme/src/WEB-INF/src/resources-importer/journal/templates/text_Paragraph_With_Header/text_Paragraph_With_Header.ftl

```diff
@@ -0,0 +1,13 @@
+<section>
+    <div class="text-center bg-${bg_Color.getData()}">
+        <h1>${header.getData()}</h1>
+        
+        <p>
+        	${paragraph_Text.getData()}
+        </p>
```

Comment:
```<p />``` tags are not needed since they can be added via the WYSIWYG editor.  If we need extra spacing, we should use a CSS class to achieve the spacing.